state/linklayerdevices.go

```diff
@@ -316,7 +330,9 @@ func removeLinkLayerDeviceOps(st *State, linkLayerDeviceDocID, parentDeviceDocID
 
 	var ops []txn.Op
 	if parentDeviceDocID != "" {
-		ops = append(ops, decrementDeviceNumChildrenOp(parentDeviceDocID))
+		localID, _ := st.strictLocalID(parentDeviceDocID)
+		ops = append(ops, decrementDeviceNumChildrenOp(localID))
+		//ops = append(ops, decrementDeviceNumChildrenOp(parentDeviceDocID))
```

Comment:
This is actually just a temporary shim. This method will disappear altogether in a following patch. I should have removed the comment line.